src/dev/ci_setup/.bazelrc-ci

```diff
@@ -5,6 +5,10 @@
 # Import and load bazelrc common settings for ci env
 try-import %workspace%/src/dev/ci_setup/.bazelrc-ci.common
 
+# Import and load bazelrc ci auth settings for BuildBuddy
+try-import %workspace%/src/dev/ci_setup/.bazelrc-ci.auth
```

Comment:
Not sure how I feel about dynamically creating this file under src. I feel better about having it under `.ci` or, better yet if we could target the home directory, or even workspace root.

Reply:
We could append to the users bazelrc.

Reply:
Do you have an idea about the `absolute` location we have defined for `$HOME`? I cannot use env vars on `try-import` but I can pass the absolute path of `$HOME` and create the auth file there.

Reply:
it looks like it is `/var/lib/jenkins`

Reply:
Alright, moved the `.bazelrc-ci.auth` into the `$HOME` folder